dali/server/daserver.cpp

```diff
@@ -454,7 +454,11 @@ int main(int argc, const char* argv[])
 
 #endif            
         if (dataPath.length())
+        {
```

Comment:
Fine really, but perhaps remove the lines above (448-449) doing this same thing for not _CONTAINERIZED so we are doing this once ?

Reply:
ah, I had, but missed it when I copied changes over to make this PR.
Thanks, will re-add.

Reply:
Done, squashed in.